Electron/src/Core/components/Cameras.tsx

```diff
@@ -39,20 +39,15 @@ interface IProps {
 
 interface IState {
   currentCamera: number
-  baseAddress: string[]
+  cameraIps: string[]
 }
 
 class Cameras extends Component<IProps, IState> {
   constructor(props: any) {
     super(props)
     this.state = {
       currentCamera: this.props.defaultCamera,
-      baseAddress: [
```

Comment:
I'm confused why you made this change, doesn't it do functionally the same thing?

Reply:
The change is much more future proof. Grabbing the IPs from the manifest means if the camera IP or autonomy IP changes for any reason, we don't have to change them again in the camera file.